dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -747,6 +749,42 @@ public User updateLoggedInUserMetadata(@ApiParam(hidden = true)@Parameter(hidden
         return dbuser;
     }
 
+    @PUT
+    @Timed
+    @UnitOfWork
+    @RolesAllowed({"admin", "curator"})
+    @Path("/user/{userId}/privileges")
+    @Consumes("application/json")
+    @Operation(operationId = "setUserPrivileges", description = "Updates the provided userID to admin or curator status, ADMIN or CURATOR only", security = @SecurityRequirement(name = OPENAPI_JWT_SECURITY_DEFINITION_NAME))
+    @ApiOperation(value = "Updates the provided userID to admin or curator status, ADMIN or CURATOR only", authorizations = { @Authorization(value = JWT_SECURITY_DEFINITION_NAME) }, response = User.class, hidden = true)
```

Comment:
You should remove this; see Gary's comment about both adding OpenAPI and removing Swagger

Reply:
Removing `@ApiOperation` would enable the method to be written into the swagger.yaml file since the `hidden` field in the annotation is preventing that from happening.

Reply:
You can remove the bulk of the annotation (`@Authorization`, `response`) to avoid confusion.